src/page-multi-facility/CreateBaselineScenarioPage.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import styled from "styled-components";
+
+import { createBaselineScenario } from "../database";
+import Colors from "../design-system/Colors";
+import InputButton from "../design-system/InputButton";
+import ModalDialog from "../design-system/ModalDialog";
+
+const CreateBaselineScenarioPageContainer = styled.div``;
+
+const WelcomeText = styled.div`
+  color: ${Colors.forest};
+  font-family: "Poppins", sans serif;
+  font-size: 18px;
+  font-weight: normal;
+  line-height: 180%;
+  padding-top: 20px;
+`;
+
+const ModalFooter = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: flex-end;
+  padding: 1em 0;
+  width: 100%;
+`;
+
+const ModalContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1 1;
+`;
+
+const CreateBaselineScenarioPage: React.FC = () => {
+  const history = useHistory();
+  const handleOnClick = async () => {
+    const baselineScenarioRef = await createBaselineScenario();
```

Comment:
Congratulations, you are the proud new owner of your very own Baseline Scenario! 🎉 

<img width="929" alt="Screen Shot 2020-04-17 at 3 48 03 PM" src="https://user-images.githubusercontent.com/25503/79608489-1c4ce180-80c3-11ea-9b7d-2a4d24603762.png">

Reply:
I'm not sure why, but when I pulled this down locally just now to test I am getting only an infinite spinner on the multi-facility page. I had pulled it down a couple of commits back just to look at the modal and that looked great, but now I've just got the spinner

I'm assuming this has something to do with the state for my user in Firebase, @sprad?

Reply:
Ohh, I think I know why that is, I need to set `loading: false` after it returns and there isn't a scenario.

Reply:
I'd assume it's because your user doesn't have any `facilities` so there's nothing to show.  I'm working on the saving facility functionality now, but if you go into the Cloud Firestore console you can add your own by following the lead on my scenario (with the id of `zxW7e4XiC5SAmA6sFpZd`)

Reply:
I just added a new commit to set loading to false after `getBaselineScenarioRef` returns, I think that should fix what you're seeing!

Reply:
Nice! Thanks, @daschi, that fixed it. I went to /multi-facility, it gave me the modal, I clicked Ok, it redirected me to the main page (which I understand to be temporary), I went back to /multi-facility and there was no modal anymore. I think that's all expected for now, right?